src/block/VanillaBlocks.php

```diff
@@ -829,6 +831,10 @@ public function getBreakTime(Item $item) : float{
 		self::register("cobblestone_stairs", new Stair(new BID(Ids::COBBLESTONE_STAIRS), "Cobblestone Stairs", $cobblestoneBreakInfo));
 		self::register("mossy_cobblestone_stairs", new Stair(new BID(Ids::MOSSY_COBBLESTONE_STAIRS), "Mossy Cobblestone Stairs", $cobblestoneBreakInfo));
 
+		$mossBreakInfo = new Info(BreakInfo::pickaxe(0.1));
```

Comment:
```suggestion
		$mossBreakInfo = new Info(BreakInfo::hoe(0.1));
```

Reply:
Why pickaxe ?

Reply:
You wrote it as pickaxe instead of hoe.

Reply:
I'm sorry I created a function on purpose hoe but I don't use it :/